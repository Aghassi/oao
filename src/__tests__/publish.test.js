/* eslint-env jest */
/* eslint-disable global-require, import/no-dynamic-require */

// import publish from '../publish';

jest.mock('../utils/shell');
jest.mock('../utils/writeSpecs');
jest.mock('../utils/git');

describe('PUBLISH command', () => {
  beforeEach(() => { jest.resetAllMocks(); });

  it('placeholder', async () => {
    // const git = require('../utils/git');
    // console.log(git.gitCurBranch.mock.calls);
  });
});
