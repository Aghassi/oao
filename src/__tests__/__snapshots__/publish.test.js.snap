// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PUBLISH command updates the changelog correctly 1`] = `
Array [
  Array [
    "test/fixtures/CHANGELOG.md",
    "## 99.99.99 (2017-1-1)

* Add awesome functionality
* Add epic feature
* Remove 1e5 bugs
",
    "utf8",
  ],
]
`;
